packages/store/internals/src/index.ts

```diff
@@ -1 +1,3 @@
+export { isAngularInTestMode } from './angular';
 export { NgxsBootstrapper } from './ngxs-bootstrapper';
+export { memoize } from './memoize';
```

Comment:
Why are we exposing `memoize` as well? What is this needed for at the moment?

Reply:
It's used by `@Selector`.

Reply:
@markwhitfeld  in v4 version, I guess it's better to rename this package

`@ngxs/store/internals` -> `@ngxs/store/common`

as Angular `@angular/common`

Reply:
It is called internals because you would not want people to use it. There would be no guarantee of no breaking changes if you use it. Mainly meant for plugins

Reply:
@arturovt exposing memoize seems unrelated to what we want from this PR.

Reply:
@markwhitfeld `isAngularInTestMode` uses `memoize` function:
```ts
export const isAngularInTestMode = memoize(_isAngularInTestMode);
```
and also `createSelector` uses `memoize`, how to solve it?

Reply:
It's fine being there, it doesn't need to be exposed in the index.ts though... Unless I'm missing something?